gossip/emitter/proposals_test.go

```diff
@@ -415,6 +428,9 @@ func TestMakeProposal_IfSchedulerTimesOut_SignalTimeoutToMonitor(t *testing.T) {
 	durationMetric.EXPECT().Update(any)
 	timeoutMetric.EXPECT().Inc(int64(1))
 
+	randaoMixer := NewMockrandaoMixer(ctrl)
+	randaoMixer.EXPECT().MixRandao(any).Return(randao.RandaoReveal{}, common.Hash{}, nil)
```

Comment:
nit: for return defaults, the `Return` part can be omitted

```suggestion
	randaoMixer.EXPECT().MixRandao(any)
```